stdlib/camlinternalFormat.ml

```diff
@@ -1730,10 +1734,13 @@ and make_float_padding_precision : type x y a b c d e f .
     ((b, c) acc -> f) -> (b, c) acc ->
     (a, b, c, d, e, f) fmt ->
     (x, y) padding -> (y, float -> a) precision -> float_conv -> x =
+  let default_float_precision fconv =
+    if fconv = Float_F then default_ocaml_float_precision
+    else default_float_precision in
```

Comment:
Thinking more about this, I don't like this local definition. It shadows a constant of the same name locally; but outside its scope, if people had to use "default float precision", they would use `default_float_precision` and their code would be likely yo be wrong. Could you, instead of defining both `default_float_precision` and `default_ocaml_float_precision` in the global scope, define only a `default_float_precision` function that pattern-matches on `fconv`?

Reply:
I agree, done.